pypeit/extraction.py

```diff
@@ -489,18 +489,21 @@ def spec_flexure_correct(self, mode="local", sobjs=None):
             trace_spat = 0.5 * (self.slits_left + self.slits_right)
             trace_spec = np.arange(self.slits.nspec)
             slit_specs = []
+            # get boxcar radius
+            box_radius = self.par['reduce']['extraction']['boxcar_radius']
```

Comment:
Are you sure we want the `boxcar_radius`?

For global, we are just extracting a few pixels at the center of the slit.
So the (previous) radius of 2 pixels seems fine.

But if you found using the `boxcar` was important, I'm fine with it.